src/doc/src/reference/manifest.md

```diff
@@ -109,6 +109,8 @@ resolve dependencies, and for guidelines on setting your own version. See the
 [SemVer compatibility] chapter for more details on exactly what constitutes a
 breaking change.
 
+This field is optional and defaults to `0.0.0`.
```

Comment:
Just a random idea. Wonder if we can have a more structural way to show the information of manifest key definition, such as how configuration doc currently looks like.

Reply:
I feel like each approach works well for some cases and not for others.  Wish we could have a code fence with links.  I feel like that could allow the best of both worlds.